publify_core/app/models/article.rb

```diff
@@ -93,6 +93,7 @@ def self.search_with(params)
 
   # FIXME: Use keyword params to clean up call sites.
   def permalink_url(anchor = nil, only_path = false)
+    return "" unless published
```

Comment:
Not sure, but is it possible that a post could be published and then unpublished? Is a permalink even relevant in that scenario?